cpp/test/Slice/errorDetection/IdentAsKeyword.err

```diff
@@ -37,16 +37,20 @@ IdentAsKeyword.ice:37: sequences can only be defined within a module
 IdentAsKeyword.ice:37: keyword 'short' cannot be used as sequence name
 IdentAsKeyword.ice:39: syntax error
 IdentAsKeyword.ice:40: 'moDule' is not defined
+IdentAsKeyword.ice:40: sequences can only be defined within a module
```

Comment:
Added all the errors which were being skipped due to the old 'syntax error' semantics.
You'll notice all these errors come directly after `type is not defined` or `... cannot be used as a type`.